api/v1alpha1/dspipeline_types.go

```diff
@@ -157,6 +161,11 @@ type ScriptConfigMap struct {
 	Key  string `json:"key,omitempty"`
 }
 
+type KfpLauncherConfigMap struct {
+	Name string `json:"name,omitempty"`
+	Data string `json:"data,omitempty"`
```

Comment:
looks like this is suposed to be a key within the configmap .data field, so let's use `Key` instead